src/JuMP.jl

```diff
@@ -738,5 +729,19 @@ include("print.jl")
 # Nonlinear-specific code
 include("nlp.jl")
 
+
+typealias _getTypes Union{NonlinearExpression,NonlinearParameter, AffExpr}
+getValue{T<:_getTypes}(arr::Array{T}) = map(getValue, arr)
```

Comment:
But that just falls back on ``$op(las, rhs[I])``, which will give a no method error.